civitai-downloader-v2/src/api/main.py

```
@@ -0,0 +1,518 @@
+            "timestamp": datetime.now(timezone.utc).isoformat()
+        }, client_id)
+    
+    def disconnect(self, client_id: str):
+        if client_id in self.active_connections:
+            del self.active_connections[client_id]
+    
+    async def send_personal_message(self, message: dict, client_id: str):
+        if client_id in self.active_connections:
+            try:
+                await self.active_connections[client_id].send_text(json.dumps(message))
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)